### PR TITLE
DO NOT MERGE: VM display help

### PR DESCRIPTION
**Pull request by cbosdo**
Documentation for PR uyuni-project/uyuni#417
Misses a screenshot to be added

### CODE REVIEW

File: adoc/reference-webui-systems.adoc [hunk 1]
```diff
@@ -1516,6 +1517,17 @@ The new virtual machine will start immediately after it has been defined.
 
 // TODO Add a screenshot of the create page showing the additional fields
 
+[float]
+[[sdc-virt-host-guest-graphical-console]]
+===== Display a virtual machine graphical console[Salt]
+
+When showing virtual machine graphical console, the user may be prompted a password.
```
Comment (Loquacity) [inline adoc/reference-webui-systems.adoc]: ``The virtual machine graphical console might prompt you for a password.``

Comment (cbosdo) [reply adoc/reference-webui-systems.adoc]: Fixed.

**Comment by Loquacity** [comment]
Would you like me to merge this for you (with the image to be added later on), or are you going to provide the image?

**Comment by cbosdo** [comment]
@Loquacity I don't have write access to the repo, thus can't merge. As for the image, I thought those were automatically generated... I'ld be happy to help adding the virtualization screenshot to the script.

**Comment by Loquacity** [comment]
Happy to merge for you. If you have an image that would be great, otherwise we can sort it out later on.

**Comment by Loquacity** [comment]
DO NOT MERGE! I will create a new PR replicating this in the /beta1 branch

**Comment by Loquacity** [comment]
For the AT component, please review: https://github.com/SUSE/doc-susemanager/pull/356
For the Reference Guide component, that part hasn't been migrated yet, will revisit once we have it in.

**Comment by Loquacity** [comment]
Closing this PR to clear the decks for rearranging the branches.

Tracking in https://github.com/SUSE/spacewalk/issues/7541

**Pull request closed without merging**